fcrypt/fcrypt.go

```diff
@@ -220,6 +228,19 @@ func (handler *CryptoHandler) GetTLSConfig() (cfg *tls.Config, err error) {
 	return cfg, nil
 }
 
+// DecryptAndValidate decrypts and validates encrypted image.
+func (handler *CryptoHandler) DecryptAndValidate(encryptedFile, decryptedFile string, params DecryptParams) error {
+	if err := handler.decrypt(encryptedFile, decryptedFile, &params); err != nil {
+		return err
+	}
+
+	if err := handler.validateSigns(decryptedFile, &params); err != nil {
```

Comment:
in case of validate error should we clena p decrypted flie here?

Reply:
Done

Reply:
Why not remove the file in case of error? It is obvious.

Reply:
Done